packages/testing/src/injectable_test.ts

```diff
@@ -0,0 +1,41 @@
+import { SetMode } from '@seagull/mode'
+import { Container } from 'inversify'
+
+export abstract class InjectableTest {
+  abstract injectDiModules: any[]
+  abstract inject: any[]
+  injector = new Container()
+
+  /**
+   * before every test, activate all given mocks
+   */
+  before() {
+    new SetMode('environment', 'pure').execute()
+    this.injector = new Container()
+    for (const diMod of this.injectDiModules) {
+      this.injector.load(diMod)
+    }
+    for (const injectable of this.inject) {
+      this.injector.bind(injectable).toSelf()
+    }
+    if (typeof (this as any).beforeEach === 'function') {
+      ;(this as any).beforeEach()
+    }
+    return 'key'
+  }
+
```

Comment:
/** Hook that is to be executed after each and every single test without compromizing

* the logic of the abstract class InjectionTest **/
    
afterEach = noop